src/Evolution/Systems/GeneralizedHarmonic/Actions/NumericInitialData.cpp

```diff
@@ -28,24 +28,23 @@ void initial_gh_variables_from_adm(
     const tnsr::ii<DataVector, 3>& extrinsic_curvature, const Mesh<3>& mesh,
     const InverseJacobian<DataVector, 3, Frame::ElementLogical,
                           Frame::Inertial>& inv_jacobian) {
-  // Take numerical derivatives
+  // Assemble spacetime metric from 3+1 vars
+  gr::spacetime_metric(spacetime_metric, lapse, shift, spatial_metric);
+
+  // Compute Phi from numerical derivative of the spacetime metric so it
+  // satisfies the 3-index constraint
+  partial_derivative(phi, *spacetime_metric, mesh, inv_jacobian);
+
+  // Compute Pi by choosing dt_lapse = 0 and dt_shift = 0 (for now).
+  // The mutator `SetPiFromGauge` should be combined with this.
   const auto deriv_spatial_metric =
       partial_derivative(spatial_metric, mesh, inv_jacobian);
```

Comment:
You could, just in case, set the d_spatial_metric to the spatial components of `Phi`, but those numbers should be identical, so I don't care. Up to you!